pkg/compose/plugins.go

```diff
@@ -156,24 +164,3 @@ func (s *composeService) setupPluginCommand(ctx context.Context, project *types.
 	cmd.Env = append(cmd.Env, types.Mapping(carrier).Values()...)
 	return cmd
 }
-
-func (s *composeService) checkPluginEnabledInDD(ctx context.Context, plugin *manager.Plugin) error {
-	if integrationEnabled := s.isDesktopIntegrationActive(); !integrationEnabled {
-		return fmt.Errorf("you should enable Docker Desktop integration to use %q provider services", plugin.Name)
-	}
-
-	// Until we support more use cases, check explicitly status of model runner
-	if plugin.Name == "model" {
-		cmd := exec.CommandContext(ctx, "docker", "model", "status")
-		_, err := cmd.CombinedOutput()
-		if err != nil {
-			var exitErr *exec.ExitError
-			if errors.As(err, &exitErr) && exitErr.ExitCode() == 1 {
-				return fmt.Errorf("you should enable model runner to use %q provider services: %s", plugin.Name, err.Error())
-			}
-		}
-	} else {
-		return fmt.Errorf("unsupported provider %q", plugin.Name)
-	}
-	return nil
-}
```

Comment:
Maybe we could keep it for Model Runner and just remove the `unsupported provider` error, wdyt?

Reply:
If plugin is installed, but disabled in DD, won't it report an adequate error message ?

Reply:
```
$ docker compose up
[+] Running 1/1
 ✘ node  error                                                                                                                                                0.0s 
Failed to pull model: service unavailable
```
(model-cli could be improved to send a more user-friendly error)